documentation/src/object-Initializer.md

```diff
@@ -0,0 +1,14 @@
+# Initializer
```

Comment:
You also need to update [SUMMARY.md](https://github.com/messerli-informatik-ag/code-style/blob/main/documentation/src/SUMMARY.md) so that the chapter is included in the generated website.

Reply:
a307d749dc25dd00af008083a96e8e6b9891d827